plugins/analytics/public/censys.py

```diff
@@ -0,0 +1,42 @@
+from datetime import timedelta
+
+from core.schemas import task
+from core import taskmanager
+from core.config.config import yeti_config
+from core.schemas.observable import Observable
+from core.schemas import indicator
+from censys.search import CensysHosts
+
+
+class CensysApiQuery(task.AnalyticsTask):
+  _defaults = {
+      "name": "Censys",
+      "description": "Executes Censys queries (stored as indicators) and tags the returned IP addresses.",
+      "frequency": timedelta(hours=24),
+  }
+
+  def run(self):
+    hosts_api = CensysHosts(
+        api_id=yeti_config.get("censys", "api_key"),
+        api_secret=yeti_config.get("censys", "secret")
+        )
```

Comment:
Please check that these two config settings are set and bail with a warning (logging.warning) if they're not?